service/src/main/java/org/cbioportal/genome_nexus/service/internal/BaseVariantAnnotationServiceImpl.java

```diff
@@ -143,7 +161,14 @@ private VariantAnnotation getVariantAnnotationExternally(String variant)
         try {
             // get the annotation from the web service and save it to the DB
             variantAnnotation = Optional.ofNullable(this.resourceFetcher.fetchAndCache(normalizedVariant));
-
+            
+            // add new annotation to index db
+            variantAnnotation.ifPresent(annotation -> {
+                Gson gson = new Gson();
+                DBObject dbObject = BasicDBObject.parse(gson.toJson(this.extractIndex(annotation)));
+                this.indexRepository.saveDBObject("index", normalizedVariant, dbObject);
```

Comment:
Should we have a method for this one like `saveToIndexDb` or something and reuse it below?